reports/Oregon.md

```diff
@@ -82,6 +82,45 @@ id: or-portland-5
 **Links**
 
 * https://twitter.com/MrAndyNgo/status/1268224547272003585
+* https://twitter.com/greg_doucette/status/1268051560489193473
+
+
+### Tear gas fired across active street at protestors | (Believed to be) June 2nd
+
+In events occurring either late on June 2nd or in the early morning of June 3rd, police can be heard declaring unlawful assembly and seen firing tear gas across the street at gathered protestors. Some protestors appear to throw tear gas canisters back.
+
+There are numerous passing cars that appear unrelated to the protest but are caught in the gas. They struggle to maneuver around canisters & clouds of tear gas.
+
+tags: tear-gas
+
+id: or-portland-22
+
```

Comment:
missing link here. did something get confused with or-portland-21?

Reply:
or did the link somehow get moved up in the first line of this diff section?

Reply:
Fuck me this is going to be a nightmare to figure out